### PR TITLE
Set up the main files and classes

### LINKED ISSUE #3: [1p]-[Group Task] Add all methods for Item class in the diagram
- All Item class properties visible in the diagram should be defined and set up in the constructor method.

Exception: properties for the 1-to-many relationships should NOT be set in the constructor method. Instead, they should have a custom setter method created.

### LINKED ISSUE #4: [1p]-[Group Task] Implement methods for Item class
- Implement methods:
    - can_be_archived?() in the Item class
        - should return true if published_date is older than 10 years
        - otherwise, it should return false
    - move_to_archive() in the Item class
        - should reuse can_be_archived?() method
        - should change the archived property to true if the result of the can_be_archived?() method is true
        - should do nothing if the result of the can_be_archived?() method is false

### PR DESCRIPTION
**Pull request by duskotmane**
# Features 🌟 

- [ ] Create `main.rb` file and implement startup actions.
- [ ] Create `Item` class and implement properties
- [ ] Create `can_be_archived?` method at `Item` class.
- [ ] Create `move_to_archive` method at `Item` class.
- [ ] Create `add_genre`, `add_author`, `add_source` and `add_label` methods at `Item` class.
- [ ] Add basic tests for `Item` class.

### CODE REVIEW

File: main.rb
Comment (luigirazum) [inline main.rb:1]: - ✅ Well done creating the entry point for the App.

Thanks!

File: README.md
Comment (luigirazum) [inline README.md:1]: - ⭐ Thanks for adding the README.

**Review by luigirazum — APPROVED** [review]
STATUS: 🟢 APPROVED!!!  🎉 🎉 🎉
===
Hi @otmaneechchafyky, 🙋‍♂️ I'm Luis and I'll be your code reviewer. 
---
### Great job !!! 👏 👏 👏 
Your project is **complete!**
There is nothing else to say other than...

It's time to merge it :shipit:
Now you can merge this branch and move on 🥇 
---
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:
===
## To Highlight 🎉
- There are no Linters errors ✔️
- The code works as expected. 👏
- Professional and descriptive README.md file 💯

Congratulations-Memes-FEATURED

## Optional suggestions ✍️
- N/A
> _Every comment with the <kbd>OPTIONAL</kbd> prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

***
Cheers and Happy Coding! 👨‍💻
Happy Coding! 👨‍💻 and Good Luck 🍀 with the coming ones 🤞
---
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me `@luigirazum` in your question so I can receive the notification, you can also reach me out on Slack using the same tag as well.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._** 

----
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb [hunk 1]
```diff
@@ -64,32 +22,13 @@ def list_options
...
-    when '8'
-      list_all_sources
-    when '9'
-      add_a_book
-    when '10'
-      add_a_music_album
-    when '11'
-      add_a_game
+    when '1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11'
+      cm_app.run(number)
```
Comment (duskotmane) [inline main.rb]: this is a new syntax for me, and I really liked it 👍

**Review by duskotmane — COMMENTED** [review]
# Approved ✔️ ✔️ ✔️ 

Everything is working properly, there is nothing else to say but it's time to merge it

# Optional ☂️ 

It would be great if we add some comments when we are going to change some existing files especially if we are refactoring the code.

**Pull request merged**